packages/teleport/src/Kubes/ConnectDialog/ConnectDialog.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import Dialog, {
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogContent,
+} from 'design/Dialog';
+import { Text, Box, ButtonSecondary } from 'design';
+import TextSelectCopy from 'teleport/components/TextSelectCopy';
+import { AuthType } from 'teleport/services/user';
+
+function ConnectDialog(props: Props) {
+  const { onClose, username, authType, kubeConnectName } = props;
+  const { hostname, port } = window.document.location;
+  const host = `${hostname}:${port || '443'}`;
+
+  let text: string;
+
+  if (authType === 'local') {
+    text = `tsh login --proxy=${host} --auth=${authType} --user=${username}`;
+  }
+
+  if (authType === 'sso') {
+    text = `tsh login --proxy=${host}`;
+  }
```

Comment:
the two text's share one common base: `tsh login --proxy=${host}`, i'd extend this base if the `authType` is local

Reply:
i'd either add a snapshot test or a render and look for the specific output test

Reply:
I've changed it to a single string literal.